torchvision/transforms/v2/_augment.py

```diff
@@ -141,7 +141,7 @@ def _transform(
 
 
 class _BaseMixupCutmix(Transform):
-    def __init__(self, *, alpha: float = 1, num_classes: int, labels_getter="default") -> None:
+    def __init__(self, *, alpha: float = 1., num_classes: int, labels_getter="default") -> None:
```

Comment:
That's a bugfix - bug that I introduced myself when I added the default, and didn't catch because we never tested on the default value (hence the slightly updated test above)

Reply:
I'm ok with making the default value the right type, but should we maybe also to `self.alpha = float(alpha)` below? I feel like `MixUp(alpha=2)` is a reasonable thing to do. The user doesn't need to know that internally floats are required.